RSEAUNZAccessibility.Rmd

```diff
@@ -0,0 +1,334 @@
+  
+
+- Ensure that PowerPoint or PDF slides are created with accessibility features. 
+  MS Office has an accessibility checker. Some instructions:  
+  
+  - [Adobe](https://helpx.adobe.com/acrobat/using/create-verify-pdf-accessibility.html)  
+
+  - [Power Point](https://support.microsoft.com/en-us/office/make-your-powerpoint-presentations-accessible-to-people-with-disabilities-6f7772b2-2f33-4bd2-8ca7-dae3b2b3ef25) Use the Accessibility Checker and add alt-text before saving as PDF.
+
+
+## Giving the Talk
+
+- Speak clearly and not too quickly. 
+  Keeping your face visible will help people understand you by reading your lips or facial expression.
+  Faking eye contact is not necessary, since some presenters will find this impossible or uncomfortable.  
+  
+- Mention slide numbers or headers when switching slides. 
+  Use descriptions to help orient people following along with a screen reader.
+  
+- Avoid assuming that things are "obvious" in your slides. 
+  Mention these things aloud.
+  
+# After the Unconference
+
+The RSE Australia/New Zealand organization has the potential to both expand its membership and 
+enhance career opportunities and visibility for disabled research software engineers.
+Organizers should be aware of the chicken-and-egg problem of inclusion on all aspects of diversity,
+but particularly for people with disabilities who face specific barriers to participation. 
+We haven't been seen at conferences or in professional settings, so accessibility has been overlooked.
+Building the infrastructure to allow a variety of disabled people to attend will lead to 
+increased participation,
+but the barriers must be removed first.  
+
+To facilitate progress from year to year, it's quite useful to survey participants after the conference. 
+People are used to inaccessibility and may have learned that their feedback is
+usually ignored. 
+Asking directly about it will help convince participants that their experience matters.
+
+Keep track of the accessibility practices you used, and how to implement them. 
+Retain this information along with feedback data and organizer comments. 
+If you document the accessibility work and results, it's more likely that as members of 
+organizing committees change over the years, the knowledge is retained.
+Due in large part to a change in the organizational leadership for UseR!2022, 
+it was not as inclusive across many historically excluded groups as UseR!2021, 
+and not screen reader accessible.
+*Accessibility is fragile.* 
+To mitigate this, the Global UseR! Working Group created a 
+[Knowledgebase](https://gitlab.com/rconf/userknowledgebase) 
+as a repository for resources and lessons learned.
+
+
+One way to promote inclusion in a conference is to make sure the leadership
+and organizing committee are diverse. 
+Invite people to join rather than 
+relying on self-nomination. 
+The article referenced below provides further advice about leadership, continuity, and diversity.
```

Comment:
Is there an article that this references to?

Reply:
On line 332 - The article referenced below provides further advice about leadership, continuity, and diversity.

But there doesn't seem to be an article referenced. Sorry, just making that clearer.